ddtrace/appsec/ddwaf/__init__.py

```diff
@@ -109,18 +110,29 @@ def run(
 
             ctx = ddwaf_context_init(self._handle)
             if ctx == 0:
-                raise RuntimeError
+                LOGGER.error("DDWaf failure to create the context")
+                return DDWaf_result(None, [], 0, (time.time() - start) * 1e6)
+
+            result = ddwaf_result()
+            wrapper = ddwaf_object(data)
+            error = ddwaf_run(ctx, wrapper, ctypes.byref(result), timeout_ms * 1000)
+            if error:
+                LOGGER.warning("DDWAF error: %d", error)
```

Comment:
this all needs to be in the first `try` right?

Reply:
good catch, thanks!